client/lib/gsuite/get-google-mailbox-purchase-cost.ts

```diff
@@ -0,0 +1,7 @@
+import type { EmailCost, ResponseDomain } from 'calypso/lib/domains/types';
+
+export function getGoogleMailboxPurchaseCost( domain: ResponseDomain ): EmailCost {
```

Comment:
I think this should return `null` or `undefined` when we don't have the cost. Without that, we have no clear way of identifying when the amount is actually 0 and when we couldn't compute the cost.